### ISSUE TITLE
TLS handshake

### ISSUE THREAD

**Issue by xujiasci** [aiortc/aioquic#158]
When I using a c++ quic29 server, using this aio python client to connect to the server. Unfortunately, the server was close by the client during connection establishment because of unexpected_message. I inspect the client python code, the exception was arised in tls.py. Client recieved CERTIFICATE_REQUEST message when client's TLS state was in CLIENT_EXPECT_CERTIFICATE_REQUEST_OR_CERTIFICATE . I notice there is a FIXME annotation, it means that is a known issue. I wonder when you decide to fix it?

**Comment by xujiasci** [comment]
By disable server send CERTIFICATE_REQUEST , I work around this

**Comment by jlaine** [comment]
Right, we don't support client certificates for the moment. I did not get round to implementing this for lack of a server against which to test.

Do you have a public server I can test against, and a client certificate I can use?

**Comment by jlaine** [comment]
@xujiasci ping?

**Comment by xybu** [comment]
Some C++ implementations of QUIC have cert validation features. For example Facebook's QUIC implementation (mvfst) can be configured to require a client cert -- https://github.com/facebookincubator/mvfst/tree/main/quic/server

Server cert and verify requirements can be configured in FizzServerContext
* https://github.com/facebookincubator/mvfst/blob/707cbf8d7d44f289345490ed19271eb64fa03032/quic/common/test/TestUtils.cpp
* See ClientAuthMode in FizzServerContext -- https://github.com/facebookincubator/fizz/blob/3c0cb003df2e71639b22eb1cefd171fa75c1a067/fizz/server/FizzServerContext.h

If no public quic server is available, maybe give that a try?

**Comment by jlaine** [comment]
Hi, do you plan to provide / set up a public server I can use or should we close this issue?
